src/variable.jl

```diff
@@ -410,6 +410,98 @@ end
 
 SymbolicIndexingInterface.getname(x, val=_fail) = _getname(unwrap(x), val)
 
+function SymbolicIndexingInterface.symbolic_evaluate(ex::Union{Num, Arr, Symbolic, Equation, Inequality}, d::Dict; kwargs...)
+    fixpoint_sub(ex, d; kwargs...)
+end
+
+function fixpoint_sub(x, dict; operator = Nothing)
+    y = fast_substitute(x, dict; operator)
+    while !isequal(x, y)
+        y = x
+        x = fast_substitute(y, dict; operator)
+    end
+
+    return x
+end
+
+const Eq = Union{Equation, Inequality}
+function fast_substitute(eq::Eq, subs; operator = Nothing)
```

Comment:
document?

Reply:
Fixed in https://github.com/JuliaSymbolics/Symbolics.jl/pull/1089/commits/33ce32b6304e22a59f0df54484bb75be93e94dec